lib/tasks/evm_test_helper.rb

```diff
@@ -1,5 +1,6 @@
 module EvmTestHelper
-  VMDB_SPECS        = FileList["spec/**/*_spec.rb"].exclude(/^spec\/(replication|migrations|automation)/)
+  #TODO: Reuse the array of excluded directories for building the FileList exclusion
+  VMDB_EXCLUDED_SPEC_DIRECTORIES = %w(replication migrations automation).freeze
```

Comment:
Of the many things I tried including tagging, providing a list of spec directories was the simplest:

`./spec/models ./spec/controllers ...`
parallel_tests knows how to handle these correctly and give the work to rspec in parallel processes.